fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,22 +57,56 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      requestOpenRollCall(event.idAlias).catch((e) =>
-        console.debug('Unable to send Roll call re-open request', e),
-      );
+      const time = Timestamp.EpochNow();
+      requestReopenRollCall(event.idAlias, time)
+        .then(() => {
+          if (!event.idAlias) {
+            console.debug(
+              'Unable to send roll call re-open request, the event does not have an idAlias',
+            );
+            return;
+          }
+          navigation.navigate(STRINGS.roll_call_open, {
+            rollCallID: event.id.toString(),
+            updateID: Hash.fromStringArray(
+              EventTags.ROLL_CALL,
+              lao.id.toString(),
+              event.idAlias.toString(),
+              time.toString(),
+            ),
```

Comment:
By computing the `updateID` here, you've effectively shifted responsibility for building a correctly formatted network message from the network-related code to a GUI component. I would strongly suggest to keep a clean separation between the two.